### PR TITLE
Owners: Add SIG Network leads as approvers.

### PR DESCRIPTION
**Pull request by ameukam**
InGate is a subproject of the SIG Network. This commit ensures the SIG
leads can approve pull requests.

### CODE REVIEW

**Comment by ameukam** [comment]
cc @kubernetes-sigs/ingate-maintainers

**Review by strongjz — APPROVED** [review]
/lgtm

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  sig-network-leads:
+    - aojea
+    - danwinship
+    - mikezappa87
+    - shaneutt
+    - thockin
+
+  ingate-maintainers: 
+    - cpanato
+    - gacko
```
Comment (Gacko) [inline OWNERS_ALIASES:15]:
```suggestion
    - Gacko
```

**Review by Gacko — CHANGES_REQUESTED** [review]

**Comment by Gacko** [comment]
@k8s-ci-robot is complaining about our allowed merge types. I disabled `merge` as we had issues with outdated PRs in Ingress NGINX. Any idea where to configure the preferred merge type for @k8s-ci-robot?

File: OWNERS [hunk 2]
```diff
@@ -1,9 +1,5 @@
 # See the OWNERS docs at https://go.k8s.io/owners
```
Comment (Gacko) [inline OWNERS:1]:
```suggestion
# See the OWNERS docs: https://www.kubernetes.dev/docs/guide/owners
```

File: OWNERS_ALIASES [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
```
Comment (Gacko) [inline OWNERS_ALIASES:1]:
```suggestion
# See the OWNERS docs: https://www.kubernetes.dev/docs/guide/owners
```

**Review by Gacko — CHANGES_REQUESTED** [review]

**Review by Gacko — APPROVED** [review]

**Comment by ameukam** [comment]
> @k8s-ci-robot is complaining about our allowed merge types. I disabled `merge` as we had issues with outdated PRs in Ingress NGINX. Any idea where to configure the preferred merge type for @k8s-ci-robot?

Answered in https://github.com/kubernetes/test-infra/pull/33906

**Comment by Gacko** [comment]
Yeah, I remembered there was something in the `test-infra` repo for `ingress-nginx`, so a quick search for `ingress-nginx` and `squash` did the trick. Thank you for approving it so fast! 🙂

**Pull request merged**